app-utils/nuclei/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=nuclei
+PKGDES="template-based vulnerability scanner"
+PKGSEC=utils
+PKGDEP="glibc"
+BUILDDEP="go"
```

Comment:
Would be nice to have an empty line before the comment, just to make the code more readable.